frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,68 @@
+import home from '#assets/img/home.svg';
+import logo from '#assets/img/logo.svg';
+import { Link } from '#libs/components/components.js';
+import { AppRoute } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useLocation } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+const Sidebar: React.FC = () => {
+  const iconSelected = 'icon-selected';
+  const iconStyles = 'icon-container';
+  const { pathname } = useLocation();
+
+  return (
+    <div className={styles['sidebar']}>
+      <nav className={styles['nav']}>
+        <Link to={AppRoute.ROOT}>
+          <div className={styles['image']}>
+            <img src={logo} alt="logo" />
+          </div>
+        </Link>
+      </nav>
+      <nav className={styles['nav']}>
+        <div className={styles['icons-container']}>
+          <button
+            className={getValidClassNames(
+              styles[iconStyles],
+              pathname === AppRoute.ROOT && styles[iconSelected],
+            )}
+          >
+            <Link to={AppRoute.ROOT}>
+              <div>
+                <img src={home} alt="home" className={styles['icon']} />
+              </div>
+            </Link>
+          </button>
+          <button
+            className={getValidClassNames(
+              styles[iconStyles],
+              pathname === AppRoute.SIGN_IN && styles[iconSelected],
+            )}
+          >
+            <Link to={AppRoute.SIGN_IN}>
+              <div>
+                <img src={home} alt="home" className={styles['icon']} />
+              </div>
+            </Link>
+          </button>
+          <button
+            className={getValidClassNames(
+              styles[iconStyles],
+              pathname === AppRoute.SIGN_UP && styles[iconSelected],
+            )}
+          >
+            <Link to={AppRoute.SIGN_UP}>
+              <div>
```

Comment:
let's use span instead of div here, not to nest block element inside inline element